lib/src/core/toastification_config.dart

```diff
@@ -81,22 +82,20 @@ Widget _defaultAnimationBuilderConfig(
 }
 
 /// Default margin builder for [Toastification]
-EdgeInsetsGeometry _defaultMarginBuilder(AlignmentGeometry alignment) {
+EdgeInsetsGeometry _defaultMarginBuilder(BuildContext context, AlignmentGeometry alignment) {
   final y = alignment is Alignment
       ? alignment.y
       : alignment is AlignmentDirectional
           ? alignment.y
           : 0.0;
```

Comment:
Since we have context now in this method, it will make sense to use resolve method of the Alignment class 

``` Dart 

  final y = alignment.resolve(Directionality.of(context)).y;
```